### PR TITLE
Draw larger arrow for final facing of move

### PR DESCRIPTION
**Pull request by sagnam**
Created a larger facing polygon for the arrow representing the facing the unit will have if the move is accepted.  This arrow is only rendered on the last valid step of a planned movement path, including the case where a hex was just entered.  Previous to this there was no facing arrow shown for a hex entered, including a hex jumped into.

I also noticed there were several variables in the area I was working that were not properly camel cased so I fixed those as well.

### CODE REVIEW

**Comment by SJuliez** [comment]
Changes look fine to me.

**Comment by sagnam** [comment]
I went ahead and fixed formatting on the entire SpriteStep file.  I also updated the logic to draw the final facing in all scenarios.  Addressed the formatting in BoardView1, but that file had a lot of formatting and linter issues and wasn't sure you'd be OK with such a large change.

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java [hunk 1]
```diff
@@ -64,35 +63,39 @@ public void prepare() {
...
-        
-        AffineTransform UpDownOffset = new AffineTransform();
-        UpDownOffset.translate(-30,0);
-        
+        Shape facingArrow = bv.facingPolys[step.getFacing()];
+        Shape finalFacingArrow = bv.finalFacingPolys[step.getFacing()];
+        if (isLastStep &&
+                (step.getMovementType(true) != EntityMovementType.MOVE_ILLEGAL)) {
+            facingArrow = bv.facingPolys[step.getFacing()];
+        }
```
Comment (Windchild292) [inline megamek/src/megamek/client/ui/swing/boardview/StepSprite.java:87]: I think this is redundant.

**Comment by NickAragua** [comment]
Is there a screenshot of what this looks like?

**Comment by Windchild292** [comment]
From facebook:
https://scontent.fykz1-2.fna.fbcdn.net/v/t1.0-9/97004151_10218686461127009_8658231491198713856_n.jpg?_nc_cat=109&_nc_sid=ca434c&_nc_oc=AQlq3M6c3iQV5mxC9KPI4y4-MvigQO4gZPAWZRXkqNaKjyWck705yE4JgB4m5ZHS1fhAEJ_BybhFgCSGnRWSPPF_&_nc_ht=scontent.fykz1-2.fna&oh=90f3f9a7eebc95b686a8737b5d0ed244&oe=5EED3643

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java (hunk 1 above)
Comment (sagnamGM) [reply megamek/src/megamek/client/ui/swing/boardview/StepSprite.java:87]: Ah!

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java [hunk 2]
```diff
@@ -30,11 +30,11 @@
...
 
     private MoveStep step;
-    boolean isLastStep = false;
+    private boolean isLastStep;
     private Image baseScaleImage;
-    
-    public StepSprite(BoardView1 boardView1, final MoveStep step,
-            boolean isLastStep) {
+
+    StepSprite(BoardView1 boardView1, final MoveStep step,
```
[the remark this answers is not in the document]
Comment (sagnam) [reply megamek/src/megamek/client/ui/swing/boardview/StepSprite.java:36]: The class was already package-private so I didn't see a reason to keep the constructor public.  Is this a style for the project?

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java (hunk 1 above)
Comment (sagnam) [reply megamek/src/megamek/client/ui/swing/boardview/StepSprite.java:87]: Ooops!

**Comment by sagnam** [comment]
I'm not sure what happened with the CI build.

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (sixlettervariables) [reply megamek/src/megamek/client/ui/swing/boardview/StepSprite.java:36]: Style in a legacy project like this is a strong word. Generally speaking "when in Rome," is best, but we are not opposed to meaningful improvements in style if coupled with meaningful work.

In this very specific case I'd say we prefer explicit access modifiers on all members.

**Comment by sixlettervariables** [comment]
I poked the CI job.

File: megamek/src/megamek/client/ui/swing/boardview/StepSprite.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (sagnam) [reply megamek/src/megamek/client/ui/swing/boardview/StepSprite.java]: Done

**Pull request merged**